### PR TITLE
Use the new Iconography everywhere

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

Remove the old Svg component and use the new Iconography everywhere

### Potential Breaking Changes

- The Svg component doesn't exist anymore, icons from the iconography must be used instead.
- A lot of components that had a `svgName` prop were replaced by an `icon` prop which expects an icon component from the iconography

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "root",
-    "version": "0.0.0",
+    "version": "0.0.0-next.10",
```
Comment (aiheon) [inline package.json:3]: t'es sûr? 🤔

File: packages/react-icons/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@coveord/plasma-react-icons",
-    "version": "45.4.0",
+    "version": "46.0.2-next.0",
```
Comment (aiheon) [inline packages/react-icons/package.json:3]: not 47?

**Comment by aiheon** [comment]
Icon cards are broken in the demo 👁️‍🗨️ https://plasma.coveo.com/feature/next/layout/IconCard

**Comment by gdostie** [comment]
> Icon cards are broken in the demo 👁️‍🗨️ https://plasma.coveo.com/feature/next/layout/IconCard

They are not, the grey square is just a img placeholder

File: package.json (hunk 1 above)
Comment (gdostie) [reply package.json:3]: woops, bad merge

File: packages/react-icons/package.json (hunk 2 above)
Comment (gdostie) [reply packages/react-icons/package.json:3]: Jenkins takes care of bumping the version, not I

**Comment by FelixBlaisThon** [comment]
In the ListBox, the tag are sticking on the label a bit (if not expected) . Other than that it looks good to me https://plasma.coveo.com/feature/next/advanced/ListBox

**Review by FelixBlaisThon — APPROVED** [review]

**Comment by gdostie** [comment]
> In the ListBox, the tag are sticking on the label a bit (if not expected) . Other than that it looks good to me https://plasma.coveo.com/feature/next/advanced/ListBox

Thanks for the review, it seems already like that on master https://plasma.coveo.com/advanced/ListBox so I'm going to let it slide for this PR

**Pull request merged**